sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -6,6 +6,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('file_format')] = object.file_format
         solr_doc[Solrizer.solr_name('file_format', :facetable)] = object.file_format
         solr_doc['all_text_timv'] = object.full_text.content
+        solr_doc[Solrizer.solr_name('file_size', Solrizer::Descriptor.new(:integer, :stored);)] = object.content.size.to_i
```

Comment:
I wonder if `Solrizer::Descriptor.new(:integer, :stored)` should be a constant defined in the `GenericFileIndexingService` class, so that `stored_integer_descriptor` in the collection_behavior above can point to that constant (i.e., use `Sufia::GenericFileIndexingService::STORED_INTEGER_DESCRIPTOR`) instead of calling `Descriptor.new` again.